pkg/cmd/server/api/install/install.go

```diff
@@ -55,6 +57,10 @@ func addVersionsToScheme(externalVersions ...schema.GroupVersion) {
 			continue
 		}
 	}
+	// we additionally need to enable audit versions, since we embed the audit
+	// policy file inside master-config.yaml
+	audit.AddToScheme(configapi.Scheme)
```

Comment:
@deads2k is this the right place to install audit types, or you want it to move it somewhere else?

Reply:
> @deads2k is this the right place to install audit types, or you want it to move it somewhere else?

Can you simplify the path leading here?  I don't think we should pretend with `externalVersions` anymore.  We have a hardcoded set of things we add and we simply add them.